client/components/LearnerViewDirective.js

```diff
@@ -52,6 +54,15 @@ tie.directive('learnerView', [function() {
                     </span>
                   </p>
                 </div>
+                <div class="tie-feedback-syntax-error">
+                <a href class="tie-feedback-syntax-error-link", ng-click="toggleSyntaxErrorHint()", 
+                        ng-show="isSyntaxErrorLinkShown">
+                  {{isSyntaxErrorShown ? 'Hide error details' : 'Display error details'}}
+                </a>
+              </div>
+              <p class = "tie-feedback-error-string", ng-show="isSyntaxErrorShown">
+                {{syntaxErrorString}}
+              </p>
```

Comment:
I think using <p> adds too much white space between the lines. Can this be a span with a br or css to handle the spacing?